pillarbox-demo/src/main/res/layout/fragment_informations.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) 2022. SRG SSR. All rights reserved.
+  ~ License information is available from the LICENSE file.
+  -->
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_margin="12dp">
+
+    <TextView
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SRGSSR/pillarbox-android/blob/9e672e88664d24e42b2fbb65a69ff4507dff46b6/pillarbox-demo/src/main/res/layout/fragment_informations.xml#L12"></span>Consider making the text value selectable by specifying <code>android:textIsSelectable="true"</code></td>
    </tr>
  </tbody>
</table>